packages/dso-toolkit/components/Componenten/dropdown-button/dropdown-button.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  notes: 'Toont een dropdown met daarin een gevulde uitklapper. Gebaseerd op de Bootstrap "Dropdowns" component',
+  notes: 'Toont een dropdown met daarin een gevulde uitklapper. Gebaseerd op de Bootstrap "Dropdowns" component. Class `text-right` op de parent van of op `.dropdown` zelf zorgt voor een rechter uitlijning.',
```

Comment:
Zo heet die class niet meer. In het PR description zelf heb ik het al gecorrigeerd.